src/index.html

```diff
@@ -7,11 +7,234 @@
+        </label>
+
+        <label
+         class="labelWithMargin"
+         >
+           I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="iacc"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset
+       class="fieldset"
+       >
+        <legend>
+            Registration
+        </legend>
+
+        <label
+         class="labelWithMargin"
+         >
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            autocomplete="off"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label
+        class="labelWithMargin"
+        >
+          Password:
+          <input
+            type="password"
+            name="password"
+            autocomplete="off"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset
+       class="fieldset"
+       >
+        <legend>
+            An interesting fact about you!
+        </legend>
+        <label
+         class="labelWithMargin"
+         >
+            Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cat"
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cat"
+              value="no"
+            >
+            No
+          </label>
+        </label>
+
+        <label
+         class="labelWithMargin"
+         >
+          What is your favorit color?
+          <input
+           type="color"
+           name="impcolor"
+          >
+        </label>
+
+        <label
+         class="labelWithMargin"
+         >
+          What time do you go to bed?
+          <input
+           type="time"
+           name="userbedtime"
+          >
+        </label>
+        <label
+         class="labelWithMargin"
+         >
+          What are your favorite brand of cars?
+          <select
+            name="car"
+            required
+            multiple
+          >
+            <option value="BMV">
+                BMW
+            </option>
+            <option value="Audi">
+                Audi
+            </option>
+            <option value="Lada">
+                Lada
+            </option>
+          </select>
+        </label>
+
+        <label
+        class="labelWithMargin"
+        >
+          How do you rate our work?
+          <input
+            type="range"
+            name="rateOurWork"
+            minlength="0"
+            maxlength="10"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset
+       class="fieldset"
+       >
+        <legend>
+            Additional info:
+        </legend>
+
+        <label
+         class="labelWithMargin"
+         >
+            Comments:
+          <textarea
+            name="feedback"
+            autocomplete="off"
+          >
+          </textarea>
```

Comment:
```suggestion
        ></textarea>
```

it should fix extra spaces in textarea field

Reply:
When I put  </textarea> like u explain it gives an error